src/Harald.WebApi/Features/Connections/Domain/Model/ChannelType.cs

```diff
@@ -0,0 +1,37 @@
+using System;
+using Harald.WebApi.Domain;
+
+namespace Harald.WebApi.Features.Connections.Domain.Model
+{
+    public class ChannelType : StringSubstitutable
+    {
+        protected internal ChannelType(string value) : base(value)
+        {
+        }
+
+              
+        public static explicit operator ChannelType(String input) 
+        {
+            return new ChannelType(input);
+        }
+        
+        public static ChannelType CreateFromString(string channelType)
```

Comment:
I am really not a big fan of the static CreateXXXX method pattern. Afaik its a leftover from .NET classic 2.0 days and most people use DI containers or seperat factory classes for this to avoid bloating classes. :)

Reply:
Ahh yes ye good old' days of .Net classic
That said being this explicit also remove the option of overloading the method. 
`All CreateFromString have been replaced with Create`